src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,25 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.Arrays;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private static final String EXCAVATOR_NAME_1 = "Excavator1";
+    private static final String EXCAVATOR_NAME_2 = "Excavator2";
+    private static final String EXCAVATOR_COLOR_1 = "Blue";
+    private static final String EXCAVATOR_COLOR_2 = "Green";
+
+    @Override
```

Comment:
Don't begin class or method implementation with an empty line. It's a minor code style issue, but keeping the code clean and consistent is important for readability.